pkg/build/process-create-build.go

```diff
@@ -0,0 +1,66 @@
+package build
+
+import (
+	"bytes"
+	"encoding/json"
+	"fmt"
+	"github.com/bugsnag/bugsnag-cli/pkg/log"
+	"github.com/bugsnag/bugsnag-cli/pkg/server"
+	"github.com/bugsnag/bugsnag-cli/pkg/utils"
+	"io"
+	"net/http"
+	"strings"
+)
+
+type Payload struct {
+	ApiKey           string            `json:"apiKey,omitempty"`
+	BuilderName      string            `json:"builderName,omitempty"`
+	ReleaseStage     string            `json:"releaseStage,omitempty"`
+	SourceControl    SourceControl     `json:"sourceControl,omitempty"`
+	Metadata         map[string]string `json:"metadata,omitempty"`
+	AppVersion       string            `json:"appVersion,omitempty"`
+	AppVersionCode   string            `json:"appVersionCode,omitempty"`
+	AppBundleVersion string            `json:"appBundleVersion,omitempty"`
+}
+
+func ProcessBuildRequest(buildOptions CreateBuildInfo, endpoint string, dryRun bool) error {
+
+	buildPayload, err := json.Marshal(buildOptions)
+
+	if err != nil {
+		log.Error("Failed to create build information payload: "+err.Error(), 1)
+	}
+
+	prettyBuildPayload, _ := utils.PrettyPrintJson(string(buildPayload))
+	log.Info("Build information: \n" + prettyBuildPayload)
```

Comment:
Is this in-keeping with logging elsewhere? Feels like it might be too low level - or use debug log?

Reply:
Even though this looks like a new addition. It in fact has not changed. Only the file name has changed hence why it look s like a new addition. It was a feature that was previously requested but happy to adjust if we feel like it is no longer needed.

Reply:
Fair point - it was probably me that requested it before :) Leave it as-is for now then.